stdlib/REPL/test/repl.jl

```diff
@@ -51,7 +51,10 @@ function fake_repl(@nospecialize(f); options::REPL.Options=REPL.Options(confirm_
     repl.options = options
 
     hard_kill = kill_timer(900) # Your debugging session starts now. You have 15 minutes. Go.
+    @eval Base old_have_color = have_color
+    @eval Base have_color = false
```

Comment:
Is it possible to remove all use of `Base.have_color` in the REPL so that this is not necessary?

Reply:
We need a mechanism to disable colors. Perhaps we need to be able to `set`